<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockDetailsDialog</class>
 <widget class="QDialog" name="BlockDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>830</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Block details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="m_blockDetailsTabBar">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <property name="leftMargin">
        <number>25</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>25</number>
       </property>
       <property name="bottomMargin">
        <number>25</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="WalletGui::WalletLargeBlackTextLabel" name="m_blockTextLabel">
           <property name="text">
            <string>Block</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="WalletGui::WalletLargeBlackTextLabel" name="m_blockHashLabel">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Click to copy</string>
           </property>
           <property name="text">
            <string>HASH</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="WalletGui::CopyMagicLabel" name="m_copyLabel">
           <property name="text">
            <string>Copied!</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="6" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_blockSizeLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_heightTextLabel">
           <property name="text">
            <string>HEIGHT</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_timestampLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_heightLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_timestampTextLabel">
           <property name="text">
            <string>TIMESTAMP (UTC)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_difficultyLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_difficultyTextLabel">
           <property name="text">
            <string>DIFFICULTY</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_transactionCountLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_transactionsSizeTextLabel">
           <property name="text">
            <string>TOTAL TRANSACTIONS SIZE</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_orphanTextLabel">
           <property name="text">
            <string>ORPHAN</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_penaltyLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_orphanLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_transactionCountTextLabel">
           <property name="text">
            <string>TRANSACTIONS</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_penaltyTextLabel">
           <property name="text">
            <string>REWARD PENALTY</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_baseRewardTextLabel">
           <property name="text">
            <string>BASE REWARD</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_transactionsSizeLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_blockSizeTextLabel">
           <property name="text">
            <string>TOTAL BLOCK SIZE</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_totalTransactionsLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_baseRewardLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_feeTextLabel">
           <property name="text">
            <string>TRANSACTIONS FEE</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_feeLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_rewardTextLabel">
           <property name="text">
            <string>REWARD</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_rewardLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_totalCoinsTextLabel">
           <property name="text">
            <string>TOTAL COINS IN THE NETWORK</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_totalCoinsLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_totalTransactionsTextLabel">
           <property name="text">
            <string>TOTAL TRANSACTIONS IN THE NETWORK</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_transactionsTab">
      <attribute name="title">
       <string>Transactions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>25</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>25</number>
       </property>
       <property name="bottomMargin">
        <number>25</number>
       </property>
       <item>
        <widget class="WalletGui::WalletTreeView" name="m_blockTransactionsView">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideMiddle</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
         <attribute name="headerStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WalletGui::CopyMagicLabel</class>
   <extends>QLabel</extends>
   <header>Gui/Common/CopyMagicLabel.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletTreeView</class>
   <extends>QTreeView</extends>
   <header>Gui/Common/WalletTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletNormalGrayTextLabel</class>
   <extends>QLabel</extends>
   <header>Gui/Common/WalletTextLabel.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletNormalBlackTextLabel</class>
   <extends>QLabel</extends>
   <header>Gui/Common/WalletTextLabel.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletLargeBlackTextLabel</class>
   <extends>QLabel</extends>
   <header>Gui/Common/WalletTextLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
